k8s.gcr.io/manifests/k8s-staging-jobset/promoter-manifest.yaml

```diff
@@ -0,0 +1,51 @@
+# google group for gcr.io/k8s-staging-jobset is k8s-infra-staging-jobset@kubernetes.io
```

Comment:
@danielvegamyhre this file should move to `registry.k8s.io` instead of `k8s.gcr.io`

Reply:
so it should be registry.k8s.io/manifests/k8s-staging-jobset/promoter-manifest.yaml

Reply:
Ah ok I see. Done.